aiogram/types/inline_keyboard.py

```diff
@@ -55,16 +55,15 @@ def add(self, *args):
             self.inline_keyboard.append(row)
         return self
 
-    def row(self, *args):
+    def row(self, *buttons):
         """
         Add row
 
-        :param args:
+        :param buttons:
         :return: self
         :rtype: :obj:`types.InlineKeyboardMarkup`
         """
-        btn_array = [button for button in args]
-        self.inline_keyboard.append(btn_array)
+        self.inline_keyboard.append(list(buttons or []))
```

Comment:
This `buttons or []` is useless, when there are no buttons in `buttons`, it will be an empty tuple, so `list(())` will return `[]`

```suggestion
        self.inline_keyboard.append(list(buttons))
```